.sequelizerc

```diff
@@ -0,0 +1,7 @@
+
+const path = require('path');
+
+module.exports = {
+  'config': path.resolve('server', 'config', 'config.json'),
+  'migrations-path': path.resolve('server', 'migrations')
+};
```

Comment:
More docs on [sequelizerc](https://sequelize.org/docs/v6/other-topics/migrations/#the-sequelizerc-file). Normally it defaults to root-level directories, this allows us to specify the directory that contains our config/migrations.